packages/block-library/src/column/block.json

```diff
@@ -9,9 +9,7 @@
 			"type": "string"
 		},
 		"width": {
-			"type": "number",
-			"min": 0,
-			"max": 100
+			"type": "string"
```

Comment:
It also creates some issues when adding a new column because it still uses number values. See https://github.com/WordPress/gutenberg/issues/27756.